### ISSUE TITLE
Dragging "live" slider fails to update value on handleEnd

### ISSUE THREAD

**Issue by mikrogroove** [andreruffert/rangeslider.js#154]
I've got a typical "progress bar" type rangeslider for a media player. The value of the field it is tied to is updated periodically by the media player while it is playing. This makes the rangeslider update it's position accordingly, but if I then drag the slider to a new position and do not drop it before it receives an update from the player, it will jump to the position of the player. By modifying the handleDown and handleEnd methods to set/unset a "sliding" flag on the object, and then testing for this in the update() function, I can stop the jump from happening, but I'm still having issues with the value sometimes not being set when the mouse is released, instead jumping to the position supplied by the player. I suspect this is a race condition with the update event being triggered simultaneously by the mediaplayer; the update event doesn't have time make the mediaplayer skip to the new position before the player sends an update in the other direction. 

What is the recommended way to use rangeslider.js with a field that is updated periodically from some source, while retaining the ability to drag it to a new value? Can I detect that the slider is being dragged from outside the control?

**Comment by andreruffert** [comment]
I guess it's a bit late I'm very busy atm...
I've made a simple [CodePen](http://codepen.io/andreruffert/full/MwXezm/) how I would handle such a use case.
:v:

**Comment by mikrogroove** [comment]
Many thanks for this, it's close to what I needed and triggered me to have another go. In the end I opted to modify the `handleDown` and `handleEnd` methods in rangeslider.js to set and unset a class "handled" on the original input element. I did not want to make my player dependent on rangeslider.js and this way can simply check if the input `.is(".handled")` when deciding whether to update the value or not.
